swagger/routes/testEvent.yaml

```diff
@@ -0,0 +1,91 @@
+/{version}/{destination}:
+  get:
+    tags:
+      - Miscellaneous
+    summary: Events Tester
+    parameters:
+      - in: path
+        name: version
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: destination
+        required: true
+        schema:
+          type: string
+    requestBody:
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              events:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    message:
+                      type: object
+                    metadata:
+                      type: object
+                    destination:
+                      type: object
```

Comment:
add the components for these respective schema

Reply:
This file is redundant and not being referenced anywhere. I have removed it.